framework/test/jest/unit/specs/modules/chain/dpos/round_delegates.js

```diff
@@ -0,0 +1,75 @@
+const BigNum = require('@liskhq/bignum');
```

Comment:
this file needs license header.
Also, the file name should be `round_delegates.spec.js`?

Reply:
I guess it's not spec because it's utility file

Reply:
ah sorry my bad